src/gsi/read_prepbufr.f90

```diff
@@ -2044,28 +2050,43 @@ subroutine read_prepbufr(nread,ndata,nodata,infile,obstype,lunout,twindin,sis,&
                  usage=100._r_kind
               end if
 
+              windbiasfact=one
+
               if (sfctype) then 
                  if (i_gsdsfc_uselist==1 ) then
                     if (kx==188 .or. kx==195 .or. kx==288.or.kx==295)  &
                     call apply_gsd_sfcuselist(kx,obstype,c_station_id,c_prvstg,c_sprvstg, &
                                             usage)
-                    if (l_rtma3d) then
-                       call tll2xy(dlon_earth,dlat_earth,x_obs,y_obs,outside_obs)
-                       if ((trim(obstype)=='t' .or. trim(obstype)=='q') .and. .not.outside_obs) then
-                          call valley_adjustment(x_obs,y_obs,usage)
-                       end if
-                    end if
+                 elseif (i_gsdsfc_uselist==2 ) then
+                    if (kx==188 .or. kx==195 .or. kx==288.or.kx==295) then
+                       call apply_sfcuselist(kx,obstype,c_station_id,c_prvstg,c_sprvstg, &
+                                               dlon_earth,dlat_earth,usage)
+                       if (gustob .and. usage>=r6) then ; gustqm=r10 ; qm=gustqm ; endif
+                    endif
                  else
                     call get_usagerj(kx,obstype,c_station_id,c_prvstg,c_sprvstg, &
                                             dlon_earth,dlat_earth,idate,t4dv-toff,      &
                                             obsdat(5,k),obsdat(6,k),usage)
                  endif
+
+                 if ((l_rtma3d .or. twodvar_regional) .and. i_gsdsfc_uselist/=1) then
+                    call tll2xy(dlon_earth,dlat_earth,x_obs,y_obs,outside_obs)
+                    if ((trim(obstype)=='t' .or. trim(obstype)=='q') .and. .not.outside_obs) then
+                       call valley_adjustment(x_obs,y_obs,usage)
+                    end if
+                 end if
```

Comment:
Hi @MatthewMorris-NOAA  @ManuelPondeca-NOAA  and @hu5970 

According to the code in line 2072~2077, when running 3DRTMA (i.e. l_rtma3d is true), the valley map adjustment will be applied only for "i_gsdsfc_uselist==2" (i.e. using EMC sfc obs usage list).  
If "i_gsdsfc_uselist==1" (using GSD uselist),  valley-map adjustment would not be used. Why is it not used in this case?  Does it cause the model crash in the following updatevars step?

Thank you!

-Gang